src/canister_tests/src/framework.rs

```diff
@@ -135,14 +134,39 @@ fn get_wasm_path(env_var: String, default_path: &path::PathBuf) -> Option<Vec<u8
     }
 }
 
-/* Here are a few useful helpers for writing tests */
-pub fn install_empty_canister(env: &StateMachine) -> CanisterId {
-    env.install_canister(EMPTY_WASM.clone(), vec![], None)
-        .expect("failed to install empty canister.")
+/** The Wasm module for the current II build, i.e. the one we're testing */
+pub static STATE_MACHINE_BINARY: &str = "../../ic-test-state-machine";
+
+pub fn env() -> StateMachine {
+    let path = match env::var_os("STATE_MACHINE_BINARY") {
```

Comment:
shouldn't `state_machine_client` deal with this?

Reply:
Hm, I'm not convinced. The library should not keep where to find the binary or which `env` variable can be supplied insted...

Reply:
We can still move this code later, if you feel strongly about it.